src/ploneintranet/todo/content/todo.py

```diff
@@ -38,7 +38,14 @@ def reopen(self):
             if milestone:
                 case_state = wft.getInfoFor(workspace, 'review_state')
                 mm = IMetroMap(workspace).metromap_sequence.keys()
-                future = mm.index(milestone) > mm.index(case_state)
+                # A case could be set to a state which isn't included in the
+                # metromap e.g. on-hold, rejected. If that happens we can treat
+                # the case_state as a future state and set all open tasks to
+                # planned
+                future = (
+                    case_state not in mm or
+                    mm.index(milestone) > mm.index(case_state)
+                )
```

Comment:
@cillianderoiste Could `index()` potentially raise `ValueError` here? (i.e Are `milestone` and `case_state` guaranteed to be keys of the metromap_sequence?)